target/linux/ramips/dts/mt7628an_wavlink_wl-wn577a2.dts

```diff
@@ -0,0 +1,117 @@
+/dts-v1/;
+#include <dt-bindings/input/input.h>
+#include <dt-bindings/gpio/gpio.h>
+
+#include "mt7628an.dtsi"
+
+/ {
+	compatible = "wavlink,wl-wn577a2", "maginon,wlr-755", "mediatek,mt7628an-soc";
+	model = "WAVLINK WL-WN577A2";
+
+	chosen {
+		bootargs = "console=ttyS0,57600";
+	};
+
+	aliases {
+		led-boot = &led_wps;
+		led-failsafe = &led_wps;
+		led-running = &led_wps;
+		led-upgrade = &led_wps;
+    };
+
+	keys {
+		compatible = "gpio-keys";
+
+		reset {
+			label = "reset";
+			gpios = <&gpio 43 GPIO_ACTIVE_LOW>;
+			linux,code = <KEY_RESTART>;
+		};
+
+		wps {
+			label = "wps";
+			gpios = <&gpio 38 GPIO_ACTIVE_LOW>;
+			linux,code = <KEY_WPS_BUTTON>;
+		};
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		led_wps: wps {
+			label = "wl-wn577a2:green:wps";
+			gpios = <&gpio 4 GPIO_ACTIVE_LOW>;
+		};
+	};
+};
+
+&state_default {
+	gpio {
+        groups = "i2c", "wled_an", "gpio", "refclk", "wdt", "p0led_an";
```

Comment:
I only see i2c wdt p0led_an required?
Despite , indent needs to be fixed.

Reply:
As stated before, the dts is based on mt7628an_wavlink_wl-wn575a3.dts. I tried to find documentation on 'wled_an' and 'p0led_an' but gave up clueless, so I just kept this line unchanged. So you saying that only 'i2c' is required?

Reply:
Removed all but i2c, wdt and p0led_an, reflashed the devices and every seems to be ok. But could you please enlighten about this section of the dts. What is  'p0led_an' and 'wled_an' all about?

Reply:
you need "i2c", "wdt", "p0led_an"
This depends on the gpio numbers you are assigning in your DTS.